examples/with-javascript/src/commands/General/command-with-subcommands.js

```diff
@@ -1,32 +1,53 @@
-const { SubCommandPluginCommand } = require('@sapphire/plugin-subcommands');
-const { send } = require('@sapphire/plugin-editable-commands');
+import { Subcommand } from '@sapphire/plugin-subcommands';
+import { send } from '@sapphire/plugin-editable-commands';
 
-class UserCommand extends SubCommandPluginCommand {
+export class UserCommand extends Subcommand {
```

Comment:
```suggestion
class UserCommand extends Subcommand {
```

JavaScript doesn't have `export`